Sources/Commands/SwiftTool.swift

```diff
@@ -379,6 +379,15 @@ public class SwiftTool<Options: ToolOptions> {
         binder.bind(
             option: parser.add(option: "--destination", kind: PathArgument.self),
             to: { $0.customCompileDestination = $1.path })
+        binder.bind(
+            option: parser.add(option: "--triple", kind: String.self),
+            to: { $0.customCompileTriple = try Triple($1) })
```

Comment:
Not a part of this PR but at some point, we should probably allow taking an array of triples in order to produce fat binaries.

Reply:
Also, can we call this `--target`? CLI frontends like clang and swiftc typically use the target terminology, whereas triple is more of an internal terminology.

Reply:
Doesn’t `--target` already exist and select a _target module_ to build as opposed to a _target platform_?

It’s what I started with, and then when I realized it would need to be `--target-triple` to disambiguate. At that point the word `target` seemed redundant.

Reply:
Yup.

```shell
$ swift build --help
[...]
   --target                Build the specified target
```

Reply:
I’m still open to alternate suggestions though.

Reply:
It's an internal flag right and the term Triple is well know so I fine with both --triple or --target-triple.

Reply:
Argh, forgot about that. Alright, let's go with `--triple` I guess.